utils/trackerbot.py

```diff
@@ -307,10 +307,13 @@ def depaginate(api, result):
     }
 
 
-def composite_uncollect(build):
+def composite_uncollect(build, source=None):
     """Composite build function"""
+    if not source:
+        source = 'jenkins'
```

Comment:
String is immutable, so it will be perfectly fine to put it as a default value in the function definition.